optimisation/ScanSelectionsTree.py

```diff
@@ -60,6 +60,9 @@
 # load cut values to scan
 ptMins = inputCfg['ptmin']
 ptMaxs = inputCfg['ptmax']
+
+minMass=inputCfg['minMass']
+maxMass=inputCfg['maxMass']
```

Comment:
You should add these arguments to all the .yml files in `optimisation`. I suggest to set the values to `None` for all the files unless the D* one

Also add spaces:
```suggestion
minMass = inputCfg['minMass']
maxMass = inputCfg['maxMass']
```

Reply:
Hi @fgrosa, thank you for updating this PR!
The .yml files in the `optimisation` directory need still to be updated, to avoid crashes with the new python script.

Reply:
Thanks @fcatalan92, I missed that!